packages/analytics-js-integrations/src/integrations/Matomo/browser.js

```diff
@@ -172,14 +174,14 @@ class Matomo {
     }
 
     // Mapping Ecommerce Events
-    if (ecommerceMapping.has(trimmedEvent)) {
-      ecommerceEventsMapping(ecommerceMapping.get(trimmedEvent), message);
+    if (ecommerceMapping?.[trimmedEvent]) {
```

Comment:
@MoumitaM @saikumarrs optional chaining would work fine here or shall I use `isDefinedAndNotNull`

Reply:
Yes this should work. If possible test it once.

Reply:
Ok cool thanks! Will add a test case for this

Reply:
done